src/Api/Controllers/EmergencyAccessController.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.Api;
+using Bit.Core.Models.Api.Request;
+using Bit.Core.Models.Api.Response;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.Controllers
+{
+    [Route("emergency-access")]
+    [Authorize("Application")]
+    public class EmergencyAccessController : Controller
+    {
+        private readonly IUserService _userService;
+        private readonly IEmergencyAccessRepository _emergencyAccessRepository;
+        private readonly IEmergencyAccessService _emergencyAccessService;
+
+        public EmergencyAccessController(
+            IUserService userService,
+            IEmergencyAccessRepository emergencyAccessRepository,
+            IEmergencyAccessService emergencyAccessService)
+        {
+            _userService = userService;
+            _emergencyAccessRepository = emergencyAccessRepository;
+            _emergencyAccessService = emergencyAccessService;
+        }
+
+        [HttpGet("trusted")]
+        public async Task<ListResponseModel<EmergencyAccessGranteeDetailsResponseModel>> GetContacts()
+        {
+            var userId = _userService.GetProperUserId(User);
+            var granteeDetails = await _emergencyAccessRepository.GetManyDetailsByGrantorIdAsync(userId.Value);
+
+            var responseTasks = granteeDetails.Select(async detail =>
+                new EmergencyAccessGranteeDetailsResponseModel(detail, await _userService.TwoFactorIsEnabledAsync(detail)));
```

Comment:
Hmm, probably not something we decided. Figured it would be useful to know how safe the accounts you grant access to are.

However since you probably have a wait period that should give you ample of time to counter the request.